live-build/misc/upgrade-scripts/common.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+UPDATE_DIR=${UPDATE_DIR:-"/var/dlpx-update"}
+DROPBOX_DIR=${DROPBOX_DIR:-"/var/delphix/dropbox"}
+MDS_SNAPNAME=${MDS_SNAPNAME:-"MDS-CLONE-upgradeverify"}
+
+function die() {
+	echo "$(basename "$0"): $*" >&2
+	exit 1
+}
+
+function report() {
+	echo -e "$(date +%T:%N:%z): $1"
+}
+
+function report_progress_inc() {
+	echo "Progress increment: $(date +%T:%N%z), $1, $2"
+}
+
+function configure_apt_sources() {
```

Comment:
Alternatively, I can just remove this function, since it's now only used in a single script.

I'd rather do that, then add comments/documentation, since it only has a single consumer.